### PR TITLE
hotfix/1.9.7 5

### PR DESCRIPTION
**Pull request by waldman**

### CODE REVIEW

File: app/models/component/manifest/app_container.rb [hunk 1]
```diff
@@ -34,7 +34,6 @@ def app_resources
       def environment_settings
         env = settings.except('PORT')
                       .map { |k, v| { name: k.to_s, value: v.to_s }.with_indifferent_access }
-        env << { name: '_JAVA_OPTIONS', value: '-Xmx1280m -Xms1280m' }.with_indifferent_access
```
Comment (Arman-Gimbal) [inline app/models/component/manifest/app_container.rb:37]: The only concern what most deployed Java apps will default to once this is removed?

**Review by ilyakam — APPROVED** [review]
The changes look good to me. However, could you please update the two commit message bodies explaining…

1. What was the security issue with the `mixlib-config` gem that forced you to lock it to a certain version?
1. What problem were the `_JAVA_OPTIONS` causing that necessitated this change?

**Comment by waldman** [comment]
@ilyakam re: 
- mixlib: None Afaik... but the latest version doesn't work with Busbar Ruby version.
- JAVA_OPTIONS: it takes precedence over any other set Java argument... what in practice makes impossible to have a container use more, or even less than this hardcoded value

**Comment by ilyakam** [comment]
Great, so please run a `git rebase -i origin 1.9.7-stable`, choose to `reword` the two commits, and update the commit message body for each one noting the reasons behind the changes.

**Comment by waldman** [comment]
@ilyakam the mixlib pinning is a colateral to the security update for the dependencies... not sure that we need to be *that* granular about the changes.

**Review by Arman-Gimbal — APPROVED** [review]

**Comment by waldman** [comment]
The current state of busbar doesn't build (and we found workaround for the issue that motivated this PR/Branch)

**Pull request closed without merging**